core/engine/src/module/mod.rs

```diff
@@ -566,6 +570,20 @@ impl Module {
         promise
     }
 
+    /// Utility function that execute [`load_link_evaluate`] and then run the job queue.
+    /// This combines two operations that are often done together into a simpler
+    /// version. If you need more control over the execution, use [`load_link_evaluate`]
+    /// and [`Context::run_jobs`] separately.
+    pub fn load_link_evaluate_and_run_jobs(&self, context: &mut Context) -> JsResult<()> {
```

Comment:
Maybe it's time to find a better name for the original function. `load_link_evaluate` sounds a bit spec-y and I think something like `run` would also be fine. That would also make the name of this function a lot shorter; `run_blocking` or something like that.

Reply:
`eval_blocking`? Either way I can make the change in this PR.

Reply:
Though about something. I think people would also want to await other promises, so maybe it should be better to add an `await_` method to `JsPromise`? That sidesteps the need for a new method for the module, and you would just call

`module.load_link_resolve(context).await_(context)?`

Reply:
PTAL